### PR TITLE
Adding changes for device connection testing

### PR DESCRIPTION
**Pull request by smita6-ai**
These changes are for MAM device connection testing.

### CODE REVIEW

**Comment by smita6-ai** [comment]
@bcarld please review changes, thank you!

File: lib/vapi/ConnectionDevices.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+class ConnectionDevices:
+
+    def __init__(self, server):
+        self.server = server
+
+    def create(self, jsonRequest):
+        """ Create a new Connection script.
+
+        :param name   -- name for the new connection script
```
Comment (bcarld) [inline lib/vapi/ConnectionDevices.py]: There is no `param` called name, just jsonRequest.
I would like to see `jsonRequest` named for the purpose it serves, not what it is (e.g. `connectionInfo`, or something similar).
This applies to `update()` also.

File: lib/vapi/base.py [hunk 2]
```diff
@@ -29,6 +29,7 @@
 from vapi.FileUserKeys import FileUserKeys
 from vapi.FileUserMetadata import FileUserMetadata
 from vapi.Categories import Categories
+from vapi.ConnectionDevices import ConnectionDevices
```
Comment (bcarld) [inline lib/vapi/base.py:32]: I think that you need to add 
```
self.connection_devices = ConnectionDevices(self.server)
```
down in the line 74-89 range.

File: lib/vapi/ConnectionDevices.py (hunk 1 above)
Comment (smita6-ai) [reply lib/vapi/ConnectionDevices.py]: Changed

File: lib/vapi/base.py (hunk 2 above)
Comment (smita6-ai) [reply lib/vapi/base.py:32]: Good catch!

**Review by bcarld — APPROVED** [review]

**Pull request merged**